spec/runners/unified/test.rb

```diff
@@ -156,6 +156,8 @@ def create_entities
                   raise NotImplementedError, "Unknown event #{event}"
                 end
               end
+            else
+              subscriber = (@subscribers[client] ||= EventSubscriber.new)
```

Comment:
The subscriber is now always created, right? If so its creation should be lifted out of the branches to before line 135 and done once.